src/main/java/ch/wisv/payments/model/MollieMethodEnum.java

```diff
@@ -9,10 +9,20 @@
  */
 public enum MollieMethodEnum {
 
+    /**
+     * Bancontact (Belgium).
+     */
+    BANCONTACT("bancontact", new ExpressionBuilder("1.01815 * x + 0.3025")),
```

Comment:
Maybe add the breakdown of this calculation in the comments, so it's clearer that the mollie website is excl. VAT and this incl. VAT